subsys/net/lib/lwm2m/Kconfig

```diff
@@ -253,6 +253,24 @@ endchoice
 
 endif # LWM2M_RESOURCE_DATA_CACHE_SUPPORT
 
+config LWM2M_SEND_SCHEDULER
+	bool "LwM2M send scheduler helper objects (10523/10524)"
+	depends on LWM2M_VERSION_1_1
+	depends on LWM2M_RESOURCE_DATA_CACHE_SUPPORT
+	select CBPRINTF_FP_SUPPORT
```

Comment:
Why is that needed?

Reply:
it's for the debug information but I think I can try to remove it

Reply:
removed